### DISCUSSION TITLE
Quarkus VertX metrics not exposed on default KC metrics endpoint

### DISCUSSION THREAD

**Discussion by avanlangup** [keycloak/keycloak#22679] [Q&A]
I am interested in _http.client.queue.delay_ (see VertxHttpClientMetrics.java)

So I enable all Quarkus at KC build time (https://quarkus.io/guides/telemetry-micrometer#quarkus-micrometer_configuration)
quarkus.micrometer.binder.http-client.enabled=true
quarkus.micrometer.binder.http-server.enabled=true

However, I can see only  **http_server_requests_xxx**  (https://quarkus.io/guides/telemetry-micrometer#automatically-generated-metrics)

Does anyone know how to export Quarkus micrometer metrics to Prometheus ?

Note: for now, I use OTEL java agent. But it registers different metrics. 
So far we did not find suitable metrics to autoscale.  Any suggestion ?

**Comment by vmuzikar** [comment]
This might be related to https://github.com/keycloak/keycloak/issues/22408

**Reply by codespearhead** [reply]
@avanlangup Is this issue still reproducible on the latest major version of Keycloak (24.0.2)? Issue #22742 (duplicate of #22408) was closed a few days after this discussion was created.

**Reply by avanlangup** [reply]
we did not migrate to 24 so far
